app/controllers/queues_controller.rb

```diff
@@ -52,8 +53,38 @@ def spam_complete
   end
 
   def suggested_edit
+    @suggested_edit = SuggestedEdit.find_by(reviewed_by_id: nil)
+    if @suggested_edit.nil?
+      flash[:info] = "There are no suggested edits to review!"
+      queue_complete_redirect
+    else
+      @request = @suggested_edit.resource
+    end
   end
 
   def suggested_edit_complete
+    @suggested_edit = SuggestedEdit.find(params[:suggested_edit_id])
+    if params[:reject]
+      @suggested_edit.result = "reject"
+      @suggested_edit.reviewed_by_id = current_user.id
+    elsif params[:approve]
+      @suggested_edit.result = "approve"
+      @suggested_edit.reviewed_by_id = current_user.id
+      rescue_request = @suggested_edit.resource
+      flash[:warning] = "Edits unable to be committed. " unless rescue_request.update(@suggested_edit.new_values)
+    end
+    flash[:danger] = "Unable to save suggested edit review." unless @suggested_edit.save
+    if SuggestedEdit.all.count > 0
```

Comment:
Is this meant to be *all* suggested edits, or only *unreviewed* suggested edits?